src/app/clients/edit-client/edit-client.component.html

```diff
@@ -109,20 +121,20 @@
 
           <mat-form-field fxFlex="48%">
             <mat-label>Submitted On</mat-label>
-            <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="submittedOnDatePicker" formControlName="submittedOnDate">
+            <input matInput [min]="minDate" [max]="maxDate" [matDatepicker]="submittedOnDatePicker" formControlName="submittedOnDate" required>
             <mat-datepicker-toggle matSuffix [for]="submittedOnDatePicker"></mat-datepicker-toggle>
             <mat-datepicker #submittedOnDatePicker></mat-datepicker>
+            <mat-error *ngIf="editClientForm.controls.submittedOnDate.hasError('required')">
```

Comment:
Same for this.

Reply:
Same as above, The submitted date must be required in the Edit client view due this value was added previously